pkg/cmd/registry/artifact/crud/get/get.go

```diff
@@ -74,6 +77,7 @@ func NewGetCommand(f *factory.Factory) *cobra.Command {
 	cmd.Flags().StringVar(&opts.registryID, "instance-id", "", opts.localizer.MustLocalize("registry.common.flag.instance.id"))
 	cmd.Flags().StringVar(&opts.outputFile, "output-file", "", opts.localizer.MustLocalize("artifact.common.message.file.location"))
```

Comment:
In case there is a references flag, output-file flag is ignored. We can make it exclusive ( cmd.MarkFlagsMutuallyExclusive(..)) or just print the warning, that output file is ignored and output will be on standard output.